frontend/i18n/en.json

```diff
@@ -580,9 +580,15 @@
           "proposal_proposed": "Proposed",
           "proposal_missed": "Missed",
           "exiting": "Exiting",
-          "exited":"Exited",
-          "withdrawing":"Withdrawing",
-          "withdrawn":"Withdrawn"
+          "exited": "Exited",
+          "withdrawing": "Withdrawing",
+          "withdrawn": "Withdrawn",
+          "slashing": "Slashing",
+          "slashed": "Slashed",
```

Comment:
The translation seems present but I got this:
![Screenshot from 2024-08-02 11-41-25](https://github.com/user-attachments/assets/d97dba1e-0146-40fb-9124-a29650cf3a15)
🤔

Reply:
Strange, works for my ... are you on my branch?
![image](https://github.com/user-attachments/assets/e65cc237-aaeb-426f-9aa1-c9cb19bfa046)